pkgs/applications/misc/sdcv/utils.hpp.patch

```diff
@@ -1,10 +0,0 @@
---- a/src/utils.hpp
-+++ b/src/utils.hpp
-@@ -2,6 +2,7 @@
- #define _UTILS_HPP_
- 
- #include <string>
-+#include <cstdlib>
```

Comment:
I don't see this patch at upstream: https://github.com/Dushistov/sdcv/blob/v0.5.2/src/utils.hpp

Reply:
Travis passes.